MapboxMobileEvents/MMEEventsManager.h

```diff
@@ -16,40 +16,7 @@ NS_ASSUME_NONNULL_BEGIN
 
 /*! @brief events manager delegate */
 @property (nonatomic, weak) id<MMEEventsManagerDelegate> delegate;
-
-/*! @brief YES if metrics collection is enabled */
-@property (nonatomic, getter=isMetricsEnabled) BOOL metricsEnabled;
-
-/*! @brief YES if metrics collection is enabled in the simulator */
-@property (nonatomic, getter=isMetricsEnabledInSimulator) BOOL metricsEnabledInSimulator;
-
-/*! @brief YES if metrics collection is enabled when the app is in use */
-@property (nonatomic, getter=isMetricsEnabledForInUsePermissions) BOOL metricsEnabledForInUsePermissions;
-
-/*! @brief YES if debug logging is enabled */
-@property (nonatomic, getter=isDebugLoggingEnabled) BOOL debugLoggingEnabled;
-
-/*! @brief UserAgent base string, in RFC 2616 format
-    @link https://www.ietf.org/rfc/rfc2616.txt */
-@property (nonatomic, readonly) NSString *userAgentBase;
-
-/*! @brief SDK version, in Semantic Versioning 2.0.0 format
-    @link https://semver.org */
-@property (nonatomic, readonly) NSString *hostSDKVersion;
-
-/*! @brief SKU Identifier */
 @property (nonatomic, copy) NSString *skuId;
-
-/*! @brief Mapbox Access Token
-    @link https://account.mapbox.com */
-@property (nonatomic, copy) NSString *accessToken;
-
-/*! @brief baseURL */
-@property (nonatomic, null_resettable) NSURL *baseURL;
-
-/*! @brief accountType */
-@property (nonatomic) NSInteger accountType;
-
```

Comment:
I'm a little concerned that some of these public properties may be used in other frameworks. Should we consider deprecating?

Reply:
I meant to highlight all most of the properties in `MMEEventsManager`... that didn't work.

Reply:
The nice thing about `0.x.x` releases, is that we don't promise API stability at all…